enclave-runner/src/usercalls/mod.rs

```diff
@@ -1226,18 +1216,7 @@ impl<'tcs> IOHandlerInput<'tcs> {
             return Ok(self.alloc_fd(AsyncFileDesc::stream(stream_ext)).await);
         }
 
-        // try to connect to all socket addresses one by one
```

Comment:
This is now unnecessary since `tokio::net::TcpStream::connect()` takes care of it in tokio 0.2 (similar to `std::net::TcpStream`)

Reply:
Wow is there anything tokio 0.2 doesn't do now?